src/components/__tests__/NavBar.test.tsx

```diff
@@ -83,6 +97,13 @@ describe('NavBar', () => {
     expect(screen.queryByRole('subNav')).not.toBeInTheDocument();
   });
 
+  it('should display toggle on mobile', () => {
```

Comment:
I see a lot of differences from mobile and desktop, don't you need to include more tests?